tools/pytercpp/pytercpp.cpp

```diff
@@ -17,7 +17,7 @@ static bool PyListToVector(PyObject* listObj, vector<string>& outVector) {
     for (int i=0; i<numWords; i++){
         /* grab the string object from the next element of the list */
         strObj = PyList_GetItem(listObj, i); /* Can't fail */
-        char* str = PyUnicode_AsUTF8(strObj); //PyBytes_AsString( strObj );
+        const char* str = PyUnicode_AsUTF8(strObj); //PyBytes_AsString( strObj );
```

Comment:
We had compilation error if we don't make this change.
I guess it maybe because of the different version of C++ compiler between now and 3 years ago when this library was last updated.

Reply:
Thank you!